source/programme-talks.rst

```diff
@@ -0,0 +1,74 @@
+Programme of talks
+==================
+
+At PyCon UK 2015, we had approximately 70 talks.
+
+About 57 of the speakers were male, and the vast majority were of course
+extremely white.
```

Comment:
None of this is intended for public consumption, nor am I suggesting that it should make its way into the published site!